src/interface/src/app/stringify-map-config.pipe.ts

```diff
@@ -18,13 +18,11 @@ export class StringifyMapConfigPipe implements PipeTransform {
 
     let str: string = '';
     let labels: string[] = [];
-    if (
-      !!mapConfig.dataLayerConfig.filepath &&
-      mapConfig.dataLayerConfig.filepath.length > 0
-    ) {
-      let dataLabel = mapConfig.dataLayerConfig.display_name
-        ? mapConfig.dataLayerConfig.display_name
-        : mapConfig.dataLayerConfig.filepath;
+
+    if (!!mapConfig.dataLayerConfig.display_name &&
```

Comment:
Do we want to base this conditional off of if there is a valid display_name or if there is a valid layer? (Aka do we want the display name to appear if the layer isn't properly set?)

Reply:
it's a good question - we don't have a true way to vet that a layer's data is completely valid.  That said, if there is no display name, there's definitely no name to display.  I'm ok considering a layer invalid in this case if it lacks a name but has all other data.

With that, I'll leave a TODO in conditions.json noting that we don't have anything that properly vets whether layers (any of the 3) are complete.